Plugins/Weapon/NWScript/nwnx_weapon.nss

```diff
@@ -48,6 +56,21 @@ void NWNX_Weapon_SetWeaponIsMonkWeapon(int nBaseItem);
 // Set plugin options
 void NWNX_Weapon_SetOption(int nOption, int nVal);
 
+// Set Devastating Critical Event Script
+void NWNX_Weapon_SetDevastatingCritalEventScript(string sScript);
+
+// Get Devastating Critical Event Weapon (to use only on Devastating Crital Event Script)
+object NWNX_Weapon_GetDevastatingCritalEventWeapon();
```

Comment:
Would it maybe make more sense to just return the struct, instead of having separate function calls?
Example: https://github.com/nwnxee/unified/blob/master/Plugins/Player/NWScript/nwnx_player.nss#L109